### PR TITLE
Fix ApiDoc insert update failure

### PR DESCRIPTION
**Pull request by crudboy**
ApiDoc insert update Form field state save exception
image

### CODE REVIEW

File: src/routes/Document/components/AddAndUpdateApiDoc.js [hunk 1]
```diff
@@ -277,9 +282,16 @@ class AddAndUpdateApiDoc extends Component {
               ],
               initialValue: state
             })(
-              <Input
-                placeholder={getIntlContent("SHENYU.DOCUMENT.APIDOC.STATE")}
-              />
+              <Radio.Group buttonStyle="solid">
+                {Object.values(STATE_TYPE).map((e, i) => {
```
Comment (li-keguo) [inline src/routes/Document/components/AddAndUpdateApiDoc.js:286]: STATE_TYPE.map((e, i) => ...

**Comment by li-keguo** [comment]
Maybe you should fix the back end in sync

https://github.com/apache/shenyu/blob/master/shenyu-admin/src/main/java/org/apache/shenyu/admin/model/dto/ApiDTO.java

eg:
```java
    @NotNull
    @Min(0)
    @Max(3)
    private Integer state;

```

Thank you for your contribution～

**Review by li-keguo — APPROVED** [review]

File: src/routes/Document/components/AddAndUpdateApiDoc.js (hunk 1 above)
Comment (crudboy) [reply src/routes/Document/components/AddAndUpdateApiDoc.js:286]: I refer to the traversal method of other arrays in this file, so I am consistent with him

Comment (crudboy) [reply src/routes/Document/components/AddAndUpdateApiDoc.js]: I changed the way the array is traversed

**Comment by crudboy** [comment]
I try fix the back end in sync

**Pull request merged**